docs/component.md

```diff
@@ -273,6 +273,72 @@ Changes `<hello-world>Hi There</hello-world>` into:
 <hello-world><h1>Hi There</h1></hello-world>
 ```
 
+In the example above, the child (`<h1>Hi There</h1>`) of the component’s element
+(`<hello-world>`) is treated as its [can-component/content] and rendered
+wherever the `<content />` placeholder is provided in the component’s `view`.
+
+If a component is defined **without** a [can-component.prototype.view] property,
+it will render whatever `LIGHT_DOM` it is given. For example, a component as
+follows:
+
+```js
+Component.extend({
+	tag: "my-el",
+	events: {
+		click: function() {
+			// Fired when the <my-el> element is clicked
+		}
+	}
+});
+```
+
+…used like so:
+
+```html
+<my-el>Here’s my content!</my-el>
+```
+
+…will be rendered as:
+
+```html
+<my-el>Here’s my content!</my-el>
+```
+
+With [can-component.prototype.leakScope] enabled, the component’s ViewModel can
+provide data to the LIGHT_DOM it contains, such that components like this:
+
+```js
+Component.extend({
+	leakScope: true,
```

Comment:
@justinbmeyer I see your comment here about this only working with `leakScope`: https://github.com/canjs/can-component/pull/288#discussion_r205252878

But this definitely works without `leakScope` right now: https://codepen.io/anon/pen/wxEdyr

Is it a bug? Here’s the issue I originally filed about this: https://github.com/canjs/can-component/issues/270